src/editor/src/UI.cpp

```diff
@@ -99,7 +103,7 @@ void Hush::UI::DockSpace()
 
     // When using ImGuiDockNodeFlags_PassthruCentralNode, DockSpace() will render our background
     // and handle the pass-thru hole, so we ask Begin() to not render a background.
-    if (dockspace_flags & ImGuiDockNodeFlags_PassthruCentralNode)
+    if (dockspaceFlags & ImGuiDockNodeFlags_PassthruCentralNode)
```

Comment:
More of a personal preference (you can ignore this comment), but having braces even if it is a single-line conditional would be better, just to keep consistency across all statements and to avoid errors in case we need to expand this condition

Reply:
Solved